lib/test/cdo/test_i18n_string_url_tracker.rb

```diff
@@ -53,57 +53,58 @@ def test_instance_not_empty
   def test_log_given_no_string_key_should_not_call_firehose
```

Comment:
Should we add a unit test for the behavior if any of the new parameters are null?